src/kirby/components/segmented-control/segmented-control.component.html

```diff
@@ -11,14 +8,19 @@
     >
       {{ item.text }}
     </ion-segment-button>
+    <span *ngIf="item.badge" role="text" [attr.aria-label]="item.badge.description">
```

Comment:
Why not place the contents (properties, attributes and directives) directly on the `kirby-badge`-element?

Reply:
👍